src/socketTx.ts

```diff
@@ -60,15 +60,40 @@ export class SocketTx {
     return allowanceValue.lt(minimumApprovalAmount);
   }
 
-  private _validateSend(send: {
+  private async _validateSend(send: {
     data?: string | undefined;
     to?: string | undefined;
     from?: string | undefined;
   }) {
     if (this.userTxType === UserTxType.FUND_MOVR) {
-      const addresses = Object.values(socketAddresses[this.chainId]);
-      if (!addresses.includes(send.to)) {
-        throw new Error(`${send.to} is not a recognised socket address on chain ${this.chainId}`);
+      // Fetcg route.
+      const routeResponse = await Routes.getActiveRoute({ activeRouteId: this.activeRouteId });
+      if (!routeResponse.success) throw new Error("Error while fetching route.");
+      const route = routeResponse.result;
+
+      // Current User Tx - Bridge Name
+      const currentUserTx = route.userTxs[this.userTxIndex];
+      const bridgeStep = currentUserTx.steps.find((step) => step.type === 'bridge');
+      if (bridgeStep) {
+        const { name: protocolName } = bridgeStep.protocol;
+        if (
+          protocolName === socketConstants.bridges.PolygonBridge &&
```

Comment:
I think it would be better to use the types 
`import { Bridge, ChainId } from "@socket.tech/ll-core/constants/types";`

Reply:
Changed this to the suggested import.